SU-HSDO/suhumsci
# Hsd8 387 field cleanup

# READY FOR REVIEW

# Summary
- Delete 4 fields from code, including 2 fields entirely from database.

# Needed By (Date)
- 10/11

# Urgency
- High

# Steps to Test

1. checkout this branch
1. sync with partial `blt drupal:sync --site=archaeology --partial`
1. ensure the field "Mail Code" is deleted from person content type. 
1. ensure the "Custom HS People" view is still enabled. (deleting fields can cause a view to be disabled)
1. ensure the field "Byline" is deleted from news content type

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
